src/fmgc/src/guidance/LnavConfig.ts

```diff
@@ -32,7 +32,7 @@ export const LnavConfig = {
     /**
      * Whether to use the L:A32NX_DEBUG_TAS and L:A32NX_DEBUG_GS LVar for prediction speeds
      */
-    DEBUG_USE_SPEED_LVARS: false,
+    DEBUG_USE_SPEED_LVARS: true,
```

Comment:
Please remove this and the DEBUG_PREDICTED_PATH below... I have added a commit to exp doing that so that turn prediction isn't broken.